go.mod

```diff
@@ -1,9 +1,9 @@
 module github.com/cloudflare/circl
 
-go 1.16
+go 1.17
```

Comment:
Should this be 19?

Reply:
no, this is to announce "kind-of" support to previous versions.